packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -12,6 +12,8 @@ import {
   VideoDeviceConstraints,
 } from '@webex/internal-media-core';
 
+import {NoiseReductionEffect, VirtualBackgroundEffect} from '@webex/web-media-effects';
```

Comment:
try changing this to export here rather then import and export

Reply:
Done